manifests/params.pp

```diff
@@ -110,9 +110,15 @@
       $repo_opt = undef
       $nowarn_kernel = false
 
-      $package_cs_source_location = 'https://packages.docker.com/1.9/apt/repo'
-      $package_cs_key_source = 'https://packages.docker.com/1.9/apt/gpg'
-      $package_cs_key = '0xee6d536cf7dc86e2d7d56f59a178ac6c6238f52e'
+      unless $package_cs_source_location {
```

Comment:
You shouldn't need the unless here. These set the defaults, which are then overridden if required in the init.pp.